pkg/loki/client.go

```diff
@@ -66,20 +66,20 @@ func (client lokiClient) Query(query string) (*QueryResponse, error) {
 		return nil, errBuildUrl
 	}
 
-	resp, errGet := client.client.Get(u.String())
+	resp, errGet := client.client.Get(queryUrl.String())
 	if errGet != nil {
 		return nil, errGet
 	}
 
 	defer resp.Body.Close()
 
-	b, errReadAll := ioutil.ReadAll(resp.Body)
+	body, errReadAll := ioutil.ReadAll(resp.Body)
```

Comment:
I hadn't noticed earlier, but there is no readon to use ReadAll.

Just pass the reader to the json module and it will read as needed.